source/fundamentals/aggregation.txt

```diff
@@ -0,0 +1,192 @@
+===========
+Aggregation
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Aggregation operations process data and return computed results, modeled on the idea of :wikipedia:`unix pipelines </Pipeline_(Unix)?oldid=993313477>`.
+The :manual:`aggregation pipeline </core/aggregation-pipeline/>` can be thought of as a factory assembly line. Along the
+line are assembly stations, called :manual:`stages </meta/aggregation-quick-reference/#stages>`. Like an assembly station
+configured with specific tooling to perform its task, each ``stage`` uses
+:manual:`operator expressions </meta/aggregation-quick-reference/#operator-expressions>` to perform some operation or
+combination of operations.
+
+In a car factory, for example, one station might shape and bend the frame using a hydraulic press and a welder, another
+might affix the doors with a drill, and so on.
+
+Find and Aggregation Comparison
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using ``find`` operations, you can:
+
+- select **what** documents to return
+- select **what** fields to return
```

Comment:
I had gone back and forth on this since italics can be missed, but they are probably more appropriate here. I want to leave the emphasis on the current words as is as that's idea.